src/core/tests/type_prop/slice.cpp

```diff
@@ -643,7 +653,7 @@ TEST(type_prop, slice_v8_basic_const_inputs_MAX_MIN_INT_64_dynamic_dimensions_ne
                             Dimension(-1),
                             Dimension(-1)};
     PartialShape expected_out_shape{8,
-                                    Dimension(8, 18),
+                                    Dimension(4, 9),
```

Comment:
This test case update is just update for `-2` step testing, or the logic has changed?

Reply:
Yes, logic changed. Previous implementation ignored stop for this case.